src/App.tsx

```diff
@@ -1,61 +1,124 @@
 import './App.scss';
+import { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList, Todo } from './components/TodoList/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+const firstTodos = todosFromServer.map(todo => ({
+  ...todo,
+  user: usersFromServer.find(u => u.id === todo.userId),
+}));
 
 export const App = () => {
+  const [submitTitleError, setSubmitTitleError] = useState(false);
+  const [submitUserError, setSubmitUserError] = useState(false);
```

Comment:
```suggestion
  const [isSubmitTitleError, setIsSubmitTitleError] = useState(false);
  const [isSubmitUserError, setIsSubmitUserError] = useState(false);
```
for boolean variables: https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig